app/src/i18n/locales/en/admin.json

```diff
@@ -144,15 +144,27 @@
   "role-update-success-toast-title": "Role updated successfully",
   "frozen": "Frozen",
   "already-registered-admin": "{{email}} is already a registered admin",
-  "modules": "Modules",
```

Comment:
Oof. I don't like this! I just rebased off develop, so it's interesting that this conflict exists. I'm going to try to re-add these.